lib/errors.js

```diff
@@ -285,6 +285,9 @@ internals.annotate = function (stripColorCodes) {
                         delete ref[replacement];
                     }
                     else {
+                        if (typeof ref === 'string') {
```

Comment:
It's probably the good fix but in the wrong place. If the error is further down the parsed object, this will fail. This should be in the very 1st if of the loop.